app/schema/widgets/currency_widget.py

```diff
@@ -0,0 +1,30 @@
+from app.schema.widget import Widget
+from flask import render_template
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CurrencyWidget(Widget):
+    def __init__(self, name):
+        super().__init__(name)
+
+    def render(self, schema, state):
+        widget_params = {
+            'answer': {
```

Comment:
This answer object seems to repeat a lot. Is it possible to move it to the super class?